instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_tracer.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry'
+
+module OpenTelemetry
+  module Instrumentation
+    module GraphQL
+      module Tracers
+        # GraphQLTracer contains the OpenTelemetry tracer implementation compatible with
+        # the GraphQL tracer API
+        class GraphQLTracer < ::GraphQL::Tracing::PlatformTracing
+          self.platform_keys = {
```

Comment:
Where did we ever land with handling `execute_field` and `execute_field_lazy`? https://github.com/open-telemetry/opentelemetry-ruby/pull/410/files#r492104122 There was a suggestion at one point to handle these by adding them as events to current span context. Is there any prior art here in other graphql language instrumentations on what approach to take? fine to skip but just wanted to see where this stood

Reply:
Ah, so we add them as spans but its set to false via config by default? `https://github.com/open-telemetry/opentelemetry-ruby/pull/527/files#diff-d66041e54fddd1da2dabb3f3b3d15692430f451a6ff416e6fa80d12b53ef588aR38` I'd be curious to see what js/python/etc do here

Reply:
>There was a suggestion at one point to handle these by adding them as events to current span context. Is there any prior art here in other graphql language instrumentations on what approach to take? fine to skip but just wanted to see where this stood

I would take the position that the `execute_field` fields should not be events, as there is valuable timing that could be captured here.  Example would be if you had a query that had an `order` object and you retrieved `order { total }` the total may be a slow calculation and we would like to be able to surface that timing, I don't think an event is the right choice there.

The same is true for the other two though I'm not sure how interesting an `authorization` computation would be, but I'm sure there an exists a use case out there where the timing would be valuable.  

The only one I could maybe see possibly an event is the type resolution field, but again I could make the same argument for that as I would the `authorized` field, someone may have some interestingly slow type resolution code added, in which case we'd want to capture that timing.

Reply:
@robertlaurin alrighty, makes sense to me then with the approach taken here.

Reply:
>Ah, so we add them as spans but its set to false via config by default?

Yeah quiet by default, I think that's the better user experience.  It's also to be mindful of vendor limits when submitting traces.  

> I'd be curious to see what js/python/etc do here

Admittedly I haven't looked, the future improvement I'd like to do in this regard is allowing a mechanism to enable tracing these fields on per request basis.  Maybe through the use of GraphQL contexts, or otel context propagation.